### ISSUE TITLE
Esc stops working after exit() [in Lua]

### ISSUE THREAD

**Issue by AMcBain** [nesbox/TIC-80#2173]
**OS**: Windows
**Version**: 1.0.2164 (b09c50c)

(The version is/was the current download for Windows on the website, so it's possible this has been fixed in a newer version I could compile myself but many many not have the skills to do so.)

1. Boot in dev mode or turn on dev mode (not entirely sure this is needed)
2. Create a new Lua cart with the following:
```lua
t=0

function TIC()
  cls(0)

 if t>1000 then -- any interval seems to be fine; in code mine was originally ~16 minutes
  trace('Now press ^R')
  exit()
 end

  rect(10,10,50,50,14)
  t=t+1
end
```
3. Run the program with `^R`
4. Hit `Esc` to verify it works
5. Run the program again with `^R` and wait for it to exit
6. Hit `^R` to rerun the program from the command line
7. Hit `Esc` and verify it doesn't work anymore even if held down

Closing the window via the 'x' button does bring up the menu asking if I want to lose my unsaved changes, but hitting 'no' doesn't make `Esc` work again so the only thing I can do is select 'yes' and wish I'd saved a tad more often.

`^R` does seem to work even in the bugged state, it caused my original non-bug-test cart to visually flash.

**Comment by AMcBain** [comment]
The version is/was the current download for Windows on the website, so it's possible this has been fixed in a newer version I could compile myself but many many not have the skills to do so.

**Comment by nesbox** [comment]
fixed here 0254325534ba9481406e04579b5c2598e1565ee3
